contracts/astroport-tf-liquid-pooler/.cargo/config

```diff
@@ -0,0 +1,3 @@
+[alias]
+wasm = "build --release --lib --target wasm32-unknown-unknown"
+schema = "run --bin schema"
```

Comment:
renaming this to config.toml and modifying 'schema' by name of contract would make it consistent with main (and not get warning)